plugins/tar/receive-app

```diff
@@ -11,7 +12,10 @@ tar_receive_app() {
   if [[ ! -f "$DOKKU_ROOT/$APP/src.tar" ]]; then
     true
   else
-    dokku tar:build "$APP"
+    acquire_app_deploy_lock "$APP"
+    # shellcheck disable=SC2086
+    tar_build "$@"
+    release_app_deploy_lock "$APP"
```

Comment:
Yeah, we have `tar_build_locked_cmd`. Don't know if that can be called here instead.